action/action-util/src/main/groovy/com/powsybl/action/util/GeneratorModificationTaskExtension.groovy

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.action.util
+
+import com.google.auto.service.AutoService
+import com.powsybl.action.dsl.spi.DslTaskExtension
+import com.powsybl.commons.PowsyblException
+import com.powsybl.contingency.tasks.ModificationTask
+
+/**
+ * @author Olivier Perrin <olivier.perrin at rte-france.com>
+ */
+@AutoService(DslTaskExtension.class)
```

Comment:
A general cosmetic remark. Checktyle seems  not to be activated on Groovy file as  a lot a formatting rules are not respected in this file without any build issue.

Reply:
I removed the trailing semi-colons (unnecessary in Groovy) and added a single empty line between each method to ensure the ``EmptyLineSeparator`` rules.
Tell me if some other things bother you.